Source: Elias Pipping <pipping@exherbo.org>
Upstream: no
Reason: Out-of-source builds fail
--- alsa-lib-1.0.25/test/lsb/Makefile.am
+++ alsa-lib-1.0.25/test/lsb/Makefile.am
@@ -5,3 +5,5 @@
 
 AM_CFLAGS = -Wall -pipe
 LDADD = ../../src/libasound.la
+
+INCLUDES=-I$(top_srcdir)/include
